hw/drivers/ipc_nrf5340/src/ipc_nrf5340.c

```diff
@@ -470,6 +470,25 @@ ipc_nrf5340_available_buf(int channel, void **dptr)
     return 0;
 }
 
+uint16_t
+ipc_nrf5340_data_available_get(int channel)
```

Comment:
there is already ipc_nrf5340_available() for this

Reply:
see the last commit in this PR

Reply:
hmm fair enough,  but it could use ipc_nrf5340_shm_get_data_length() internally anyway ;)

Reply:
I'll think about it ;)